src/vsCodeConfig/settings.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
The whole "vsCodeConfig" file structure is kind of unique to the functions extension - partially because I have a lot more logic related to vs code settings. Also because I have an analagous "funcConfig" folder. I feel like this would be simpler as a "settingUtils.ts" file in the "utils" folder (In fact that's what I did [here](https://github.com/microsoft/vscode-cosmosdb/pull/1572/files#diff-1da0c03369229a5be1f98e829bf67baeR1) 😂)